src/components/Cluster/ClusterDetail/ClusterDetail.js

```diff
@@ -31,24 +68,36 @@ const ClusterDetail = ({ match, cluster, clusterId, ...rest }) => {
   return (
     <Breadcrumb
       data={{
-        title: match.params.clusterId,
+        title: clusterID,
         pathname: match.url,
       }}
     >
       <Switch>
         <Route
           path={`${match.path}/getting-started/`}
-          render={() => <GettingStarted {...rest} match={match} />}
+          render={() => <GettingStarted match={match} />}
         />
 
         <Route
           path={`${match.path}`}
           render={() => (
             <ClusterDetailView
-              {...rest}
               match={match}
               cluster={cluster}
-              clusterId={clusterId}
+              isV5Cluster={isV5Cluster}
```

Comment:
There's no benefit, really, since both `ClusterDetail` and `ClusterDetailView` are linked to the store. I didn't want to fiddle with the existing logic too much, but I can move all the data coming from the state to `ClusterDetailView`